srtcore/core.h

```diff
@@ -573,6 +573,8 @@ class CUDT
     int64_t m_llInputBW;                         // Input stream rate (bytes/sec)
     int m_iOverheadBW;                           // Percent above input stream rate (applies if m_llMaxBW == 0)
     bool m_bRcvNakReport;                        // Enable Receiver Periodic NAK Reports
+
+	bool m_bIpV6Only;                // IPV6_V6ONLY option
```

Comment:
Tabs -> spaces.

Reply:
Done.
What about updating the [contributing code style](https://github.com/Haivision/srt/blob/master/CONTRIBUTING.md#code-style)? I agree that I could have done more efforts to understand the style used but it could save time for both of us if the coding conventions are written clearly in a document. Or maybe it could be simplified by adding an automatic style control into the bot directly.